StreamChatClient_v3/Controllers/ChannelController_Tests.swift

```diff
@@ -569,3 +585,17 @@ private class TestDelegateGeneric: QueueAwareDelegate, ChannelControllerDelegate
         validateQueue()
     }
 }
+
+private extension MemberAddedEvent {
+    static func dummy(cid: ChannelId, userId: UserId) throws -> MemberAddedEvent {
+        let member = MemberPayload<NameAndImageExtraData>(user: .dummy(userId: userId),
+                                                          role: .member,
+                                                          createdAt: .unique,
+                                                          updatedAt: .unique)
+
+        let payload = EventPayload<DefaultDataTypes>(eventType: .memberAdded,
+                                                     cid: cid,
+                                                     memberContainer: .init(member: member))
+        return try .init(from: payload)
+    }
+}
```

Comment:
You can just create `TestMemberEvent: MemberEvent`, no need to use the real event types here.

Reply:
fixed in 791b4a3673a2127949ad673e06f1ae90366330f9